corehq/util/soft_assert/api.py

```diff
@@ -0,0 +1,107 @@
+from django.core.mail import mail_admins, send_mail
+from django.core.cache import cache
+from corehq.util.soft_assert.core import SoftAssert
+import settings
+
+
+def _number_is_power_of_two(x):
+    # it turns out that x & (x - 1) == 0 if and only if x is a power of two
+    # http://stackoverflow.com/a/600306/240553
+    return x > 0 and (x & (x - 1) == 0)
+
+
+def _django_caching_counter(key):
+    cache_key = 'django-soft-assert.{}'.format(key)
+    try:
+        return cache.incr(cache_key)
```

Comment:
No, don't know why that would be happening.

I tested this in a django shell on hqdb0 and hqdjango3 and both are working fine.